### PR TITLE
Provide meta/run.yaml and README.md for OpenFOAM packages

### PR DESCRIPTION
**Pull request by miha-plesko**
With this commit we first provide README for openfoam.core package and then meta/run.yaml and README for each openfoam solver package.

### CODE REVIEW

File: docker_files/recipes/openfoam.core/meta/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# OpenFOAM Core 2.4.0
+This package provides core libraries, binaries and default configuration required
+by the OpenFOAM solvers.
+
+## Usage
+You only need to require this package and the libraries become available.
+In fact, the openfoam.core package is included in solvers so you shouldn't
+need to require this package manually.
+
+Note that libraries reside inside /openfoam directory so solvers will need
```
Comment (gberginc) [inline docker_files/recipes/openfoam.core/meta/README.md]: Just checking: are you sure that libraries are inside `/openfoam`? Looking here, the libs are in `/usr/lib` while the `/openfoam` contains some OpenFOAM config files.

**Review by gberginc — COMMENTED** [review]
One minor correction in the core package, nothing crucial.

File: docker_files/recipes/openfoam.core/meta/README.md (hunk 1 above)
Comment (miha-plesko) [reply docker_files/recipes/openfoam.core/meta/README.md]: I stand corrected!

**Review by gberginc — APPROVED** [review]
Thanks @miha-plesko!

**Pull request merged**